Updating bits around application gateway's connection draining. (#43787)
Azure/azure-content: articles/application-gateway/application-gateway-faq.md

### Does Application Gateway support connection draining?

> Yes. You can set up connection draining to change members within a backend pool without disruption. For specifics visit [connection draining section of Application Gateway](https://docs.microsoft.com/azure/application-gateway/overview#connection-draining).

Reviewer: ```suggestion
Yes. You can set up connection draining to change members within a backend pool without disruption. For more information, see [What is Azure Application Gateway?](overview.md#connection-draining).
```

Author: I believe the link title should indicate the section it is targeting. Or users might not click it thinking that it would just take them to Azure Application Gateway's introduction and not the feature's specification.

Author: Taking other changes though.
